packages/manager/apps/pci-kubernetes/src/types/index.ts

```diff
@@ -2,6 +2,260 @@ import { z } from 'zod';
 import { pluginData } from '@/api/data/plugins';
 import { isBase64 } from '@/helpers';
 
+export type ContinentCode = 'AF' | 'ASIA' | 'EU' | 'NA' | 'US';
```

Comment:
Linked to my previous comment regarding types :)
```suggestion
export type TContinentCode = 'AF' | 'ASIA' | 'EU' | 'NA' | 'US';
```